rakelib/docker_debugging.rake

```diff
@@ -15,12 +15,10 @@ namespace :docker_debugging do
         Thread.send(:new) { loop {} }.join
       end
       if s.set_up_db
-        puts 'Starting db setup'
         Rake::Task['db:drop'].invoke
         Rake::Task['db:create'].invoke
         Rake::Task['db:schema:load'].invoke
-        Rails.env = 'test'
-        Rake::Task['db:schema:load'].invoke
+        Rake::Task['db:migrate'].invoke
```

Comment:
A schema load should have the most up to date schema. Why is this migrate needed?

Reply:
You are right!  This code is really only used by those of us made to use corporate windows boxes without Windows administrator rights.  We debug through docker and this enables us to have our migration(s) run during an up/down before schema.rb is updated.  It can be easier than logging into the container to manually run them.